README.md

```diff
@@ -109,8 +109,34 @@ export ETH_SIGNING_KEY=<Your ETH SIGNING KEY>
 pigeon start
 ```
 
-- Remember to run pigeon as a systemd Service! If you have a good systemd service implementation for Ubuntu, please make a PR on this README and we will add it.
-
+#### Using systemd service
+Create service file
+
+```shell script
+tee <<EOF >/dev/null /etc/systemd/system/pigeond.service
+[Unit]
+Description=Pigeon daemon
+After=network-online.target
+
+[Service]
+Environment="PALOMA_KEYRING_PASS=$PALOMA_KEYRING_PASS"
+Environment="ETH_RPC_URL=$ETH_RPC_URL"
+Environment="ETH_PASSWORD=$ETH_PASSWORD"
+Environment="ETH_SIGNING_KEY=$ETH_SIGNING_KEY"
```

Comment:
Given the default env variables that we are exporting and based on the config file in this readme, you need to add VALIDATOR environment variable as well.